blog/_posts/2020-10-27-tcp-mss-clamp.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "TCPMSS Clamping in Firewalld"
+section: Blog
+date: 2020-10-28T21:52:00
+author: Vrinda Punj
+category: Feature
+---
+## What is TCPMSS Clamping?
+The maximum segment size is defined as the largest amount of data that can be received in a single
+TCP segment. TCPMSS clamping is a feature that sets the maximum segment size used by a TCP
+session. The way that it achieves this is during the TCP 3 way handshake, a server can set
+the MSS in the outgoing TCP SYN packets signalling the maximum segment
+size of the data packets that it can receive.
+
+## Why is it needed?
+The way that TCPMSS clamping was enabled in Firewalld before the addition of this feature
+was that users could add it using direct rules. An example of enabling the TCPMSS clamp
```

Comment:
Wording here is a bit weird. "was that users could add it" maybe do "was by adding an iptables rule via direct rules".